### PR TITLE
Restored macros and added Al's changes

### PR DESCRIPTION
**Pull request by mjethanandani**
Reshad and Al, please review these set of changes, before I pull this PR in.

In particular, notice I have restored the macros in the XML file. They will be impossible to spot because of the large set of diffs, but needless to say they are in. Please use 'make' to generate the draft going forward. Yes, you need all the tools in place for you to do the build :-(

Al, I have added in your changes. The problem was with the close quote on line 838 - Somebody use '' instead of ". It is very subtle, and took quite a bit of time to spot.

### CODE REVIEW

**Comment by acmacm** [comment]
I reviewed the diffs, looks good. Thanks Mahesh!  I'm ready to merge if Reshad agrees..

**Comment by Reshad-Rahman** [comment]
Good with me too. Only thing I don't recall  is why repeat-count was moved into test-session-request.

File: src/yang/ietf-twamp.yang [hunk 1]
```diff
@@ -642,66 +660,68 @@ module ietf-twamp {
...
-              will be used by the Session-Reflector for
-              this TWAMP-Test session. The number is restricted
-              to the dynamic port range and is to be placed in
-              the Receiver Port field of the Request-TW-Session
-              message.";
+               will be used by the Session-Reflector for
+               this TWAMP-Test session. The default number is within
+               to the dynamic port range and is to be placed in
+               the Receiver Port field of the Request-TW-Session
+               message.";
```
Comment (acmacm) [inline src/yang/ietf-twamp.yang]: I'd like to add "The new well-known port number (862) MAY be used." to the description, but a new PR might create a merge conflict (?)

Comment (mjethanandani) [reply src/yang/ietf-twamp.yang]: Ok. Will add.

**Comment by acmacm** [comment]
Thanks Mahesh!

**Review by acmacm — APPROVED** [review]
Thanks Mahesh!

**Pull request merged**